force-app/main/default/lwc/chartBuilder/chartBuilder.js

```diff
@@ -129,13 +134,25 @@ export default class ChartBuilder extends LightningElement {
     return [POLARAREA_CHART_TYPE, RADAR_CHART_TYPE].includes(this.type);
   }
 
+  get isDimensionable() {
+    return [
```

Comment:
You can extract this array to avoid having to recreate it every time you invoke the getter.
Otherwise, you can just do a plain `===` check with `||`, this will be even faster.